phase5/SampleLibrary/FileReader.cs

```diff
@@ -0,0 +1,30 @@
+namespace SampleLibrary;
+
+public class FileReader
+{
+    private readonly string _path;
```

Comment:
It might be useful to change `_path` to read a new folder so it shouldn't be `private readonly`.

Reply:
each file reader is only meant for one path
if we want to read a new folder then a new file reader should be made (path is given to the fileReader's constructor)